code/AssetLib/glTF2/glTF2Importer.cpp

```diff
@@ -96,9 +96,6 @@ static const aiImporterDesc desc = {
 };
 
 glTF2Importer::glTF2Importer() :
-        BaseImporter(),
-        meshOffsets(),
-        mEmbeddedTexIdxs(),
```

Comment:
base class default c’tors are always called; `std::shared_ptr` is always default-initialized